src/app/pages/categories/management/categories-editor/pages-categories-editor.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="loading-container" *ngIf="isLoadingTransaction">
+  <mat-spinner [diameter]="30"></mat-spinner>
+</div>
+
+<form [formGroup]="form" class="form__container" *ngIf="!isLoadingTransaction">
```

Comment:
After changing type of the form in TS file I think it will be better to use here `*ngIf="form"` so you do not have to use something like `form?.controls.type` below.